tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -123,9 +157,39 @@ _root_.scala.Int#`+`(I)I. => abstract def +: (x: Int): Int
   [10..13): Int => _root_.scala.Int#
 _root_.scala.Int#`<init>`()V. => primaryctor <init>: (): Int
   [4..7): Int => _root_.scala.Int#
+_root_.scala.collection.GenTraversableLike#headOption()Lscala/Option;. => abstract def headOption: Option[A]
```

Comment:
Maybe we could rewrite this test to avoid having overrides.

Reply:
I think there is something wrong with overrides implementation if we are getting denotations for overrides of all referenced symbols. We should only persist the override denotation for symbols that are overridden in the document.

Reply:
Yep, make sense. I'm adding this test to the `OverridesSuite`

```scala
  overrides(
    "object a { Array.empty[Int].headOption }",
    ""
  )
```